CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ project(revng-qa NONE)
 install(PROGRAMS revng-test-configure boolparser.py common.sh
         DESTINATION libexec/revng)
 
-install(DIRECTORY share/ DESTINATION share/)
+install(DIRECTORY share/ DESTINATION share/ USE_SOURCE_PERMISSIONS)
```

Comment:
@aleclearmind On the topic of scripts, is this the right way to enable them in the `share` directory. This is what `revng`-side does, but I'm not sure if just this is enough.

Reply:
Works.